tests/bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
But then again I am very new to this land of formal dependency management in PHP :) Perhaps I fixed something that made autoloading work on my machine without capturing it in version control for others?